rotkehlchen/db/upgrades/v36_v37.py

```diff
@@ -49,6 +50,37 @@ def _reset_decoded_events(write_cursor: 'DBCursor') -> None:
     )
 
 
+def _move_event_locations(write_cursor: 'DBCursor') -> None:
+    """
+    Create location ethereum and optimism and move the blockchain events to those locations
```

Comment:
Mention in the docstring this used to be in migration 9, which happened from ... 1.26.3 to 1.27.0? Or 1.27.0 to 1.27.1?

Also mention this should be the very first part of this DB upgrade as it relied on the old schema